src/Domain.cpp

```diff
@@ -729,14 +735,29 @@ unsigned long Domain::getglobalNumMolecules() const { return _globalNumMolecules
 void Domain::setglobalNumMolecules(unsigned long glnummol) { _globalNumMolecules = glnummol; }
 
 void Domain::updateglobalNumMolecules(ParticleContainer* particleContainer, DomainDecompBase* domainDecomp) {
-	CommVar<uint64_t> numMolecules;
-	numMolecules.local = particleContainer->getNumberOfParticles(ParticleIterator::ONLY_INNER_AND_BOUNDARY);
-	domainDecomp->collCommInit(1);
-	domainDecomp->collCommAppendUnsLong(numMolecules.local);
-	domainDecomp->collCommAllreduceSum();
-	numMolecules.global = domainDecomp->collCommGetUnsLong();
-	domainDecomp->collCommFinalize();
-	this->setglobalNumMolecules(numMolecules.global);
+		bool bVal = false;
```

Comment:
Can we please change the name to something meaningful? Like e.g. `bNumPrtlsChangedGlobal`?